fmriprep/workflows/bold/base.py

```diff
@@ -539,48 +528,69 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
                     ('outputnode.syn_bold_ref', 'inputnode.in_post')]),
             ])
 
-    # if multiecho data, select first echo for hmc correction
+    # if multiecho data, process all echos but select first echo for hmc correction
     if multiecho:
-        inputnode.iterables = ('bold_file', bold_file)
+        from .util import init_skullstrip_bold_wf
+        inputnode.inputs.bold_file = ref_file  # Replace reference w first echo
+
+        if run_stc:
+            workflow.disconnect([
+                (bold_reference_wf, bold_stc_wf,
+                    [('outputnode.bold_file', 'inputnode.bold_file')])])
+            meepi_echos = boldbuffer.clone(name='meepi_echos')
```

Comment:
Is there a way to pass iterables to `LiterateWorkflow` directly ?

Reply:
Pinging @effigies as the nipype guru :)

Reply:
Treat `LiterateWorkflow` the same as any other `Workflow`. Or is the question how to pass iterables to any workflow?

Reply:
Yes, please.

Reply:
Resolved in person -- the used approach is best for now.